### PR TITLE
User story 1v2

### LINKED ISSUE #1: User Story 1
Image

As a merchant,
When I visit my merchant dashboard (/merchants/:merchant_id/dashboard)
Then I see the name of my merchant

### PR DESCRIPTION
**Pull request by dani-wilson**
# Describe the changes below:
Add tests for merchant dashboard page, as well as views, and change controller slightly.

# Relevant user story:
User story 1

# Specific feedback requests? Include file name and line:
Is it legal to have what's typically a show action in an index? I wasn't sure how else to go about this. 

# Before submitting, check the following:
- [X] Entire test suite is passing
- [100%] SimpleCov test percentage

# Note any co-authors / anything else you'd like to add:
close #1

### CODE REVIEW

**Review by MatthewTLim — APPROVED** [review]
Looks good! just a heads up there are some merge conflicts on this one.

File: spec/features/dashboard/index_spec.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    @m3 = Merchant.create!(name: "Willms and Sons")
+    @m4 = Merchant.create!(name: "Cummings-Thiel")
+    @m5 = Merchant.create!(name: "Williamson Group")
+  end
+  it "displays the name of each merchant" do
+    visit "/merchants/#{@m1.id}/dashboard"
+
+    expect(page).to have_content(@m1.name)
+    expect(page).to_not have_content(@m2.name)
+  end
```
Comment (awiley33) [inline spec/features/dashboard/index_spec.rb:16]: nice testing!

File: app/controllers/dashboard_controller.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class DashboardController < ApplicationController
+  def index
+    @merchant = Merchant.find(params[:merchant_id])
```
Comment (awiley33) [inline app/controllers/dashboard_controller.rb:3]: AR query looks good! I still don't understand why it's merchant_id but if it's working that's great!

**Review by awiley33 — APPROVED** [review]
Looks good! Nice work.

**Pull request merged**